config/hexa_control_odomless.yaml

```diff
@@ -0,0 +1,8 @@
+# Serial
+SerialPort: /dev/ttyACM0
+
+# Baudrate: 1 - 1000000, 16 - 115200, 34 - 57600
+SerialBaudrate: 1
+
+# Odometry Topic
+OdomEnable: false
```

Comment:
Right now we have many config files (one for each case). This is bad, but we discussed it with Dorian and when I have the `hexapod_ros` repo, the `resibots_launch` will be deprecated (deleted). So, I think for now is ok and we should discuss how to do it properly in the next meeting.